src/app/(pages)/pool/new/use-create-pool.ts

```diff
@@ -200,7 +200,7 @@ export function useCreatePool() {
                 })
                 .finally(() => {
                     setPoolUpdated(false)
-                    setTransactionProcessed(false)
+                    setTransactionProcessed(false) 
```

Comment:
this whitespace is not needed, normally if you format the file it will be removed by prettier

Reply:
fixed